src/bojack/commands/command.cr

```diff
@@ -0,0 +1,19 @@
+require "../memory"
+
+module Bojack
+  module Commands
+    # Represents a Bojack command
+    #
+    # It is a interface that each command
```

Comment:
Crystal supports `abstract` types. It this the case where you are enforcing users to have the `execute` method defined?
Reference: https://crystal-lang.org/docs/syntax_and_semantics/virtual_and_abstract_types.html